src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/how-to-use-content-redaction.md

```diff
@@ -13,31 +13,107 @@ toc: true
 tree_item_index: 100
 ---
 
-Protecting and removing personal identifying information (PII) from your documents can be a requirement before sharing your content to a wider audience or in order to comply with regulations such as the General Data Privacy Regulation (GDPR) or the California Consumer Privacy Act (CCPA).
+Protecting and removing personal identifying information (PII) from your documents can be a requirement in order to comply with regulations such as the General Data Privacy Regulation (GDPR) or the California Consumer Privacy Act (CCPA).
 
-Nuxeo Enhanced Viewer helps you with this use case by allowing to apply temporary or permanent redactions to your content. This page explains what to expect from this feature, and how to configure it in your Nuxeo instance.
+Nuxeo Enhanced Viewer helps you with this use case by allowing to apply temporary or permanent redactions to your content. This page explains what to expect from this feature, and how to leverage it in your Nuxeo instance.
+
+## Enabling the Redact Permission
+
+Since Nuxeo Enhanced Viewer (NEV) 2.2.0, a new permission called `Redact` is available. This permission allows a user to get access to the redaction actions described below.
```

Comment:
I'm updating _Since Nuxeo Enhanced Viewer (NEV) 2.2.0_ because technically the permission is on the Nuxeo ARender addon 10.8.0, tell me if it is not correct.

Reply:
Thanks Kevin :slightly_smiling_face: 
Added the list of components in the release notes page as it will be helpful to customers; please let me know or correct if you spot a mistake :wink:

Reply:
Table is good 👍.

Since NEV 2.x, the table is not 100% reflecting the compatibility possibilities between NEV and the addon imho.
For example all NEV 2.x is compatible with addons 10.7.y, 10.8.x, 2021.2.y, 2021.3.y. NEV 2.2 requires addons 10.8.y / 2021.3.y for the content redact feature but not to work in the same way than NEV 2.1.

But I don't have idea how to reflect that, if you have any idea?

Still this is not a big big point, we can go with the current table.